nash-native-client/src/ws_client/client.rs

```diff
@@ -49,7 +49,6 @@ pub fn spawn_heartbeat_loop(client_id: u64, outgoing_sender: UnboundedSender<Abs
 pub fn spawn_sender_loop(
     mut websocket: WebSocket,
     mut ws_outgoing_reciever: UnboundedReceiver<AbsintheWSRequest>,
```

Comment:
I am not completely familiar with the code, so I am not yet in a good position to give any opinion about it.

It seems syntactically good and I don't see any issue with it besides the typo in `receiver`.